test/steps/configuration_steps.go

```diff
@@ -25,4 +46,65 @@ func ConfigurationSteps(suite *godog.Suite, fs *FeatureState) {
 		fs.activeScenarioState.gitEnvironment.DeveloperRepo.Configuration().SetNewBranchPush(b, false)
 		return nil
 	})
+
+	suite.Step(`^the main branch is configured as "([^"]+)"$`, func(name string) error {
+		fs.activeScenarioState.gitEnvironment.DeveloperRepo.Configuration().SetMainBranch(name)
+		return nil
+	})
+
+	suite.Step(`^the main branch is now configured as "([^"]+)"$`, func(name string) error {
+		actual := fs.activeScenarioState.gitEnvironment.DeveloperRepo.FreshConfiguration().GetMainBranch()
+		if actual != name {
+			return fmt.Errorf("expected %q, got %q", name, actual)
+		}
+		return nil
+	})
+
+	suite.Step(`^the main branch name is not configured$`, func() error {
+		fs.activeScenarioState.gitEnvironment.DeveloperRepo.Configuration().DeleteMainBranchConfiguration()
+		return nil
+	})
+
+	suite.Step(`^the perennial branches are not configured$`, func() error {
+		fs.activeScenarioState.gitEnvironment.DeveloperRepo.Configuration().DeletePerennialBranchConfiguration()
+		return nil
+	})
+
+	suite.Step(`^the perennial branches are configured as "([^"]+)" and "([^"]+)"$`, func(branch1, branch2 string) error {
+		fs.activeScenarioState.gitEnvironment.DeveloperRepo.Configuration().AddToPerennialBranches(branch1, branch2)
+		return nil
+	})
+
+	suite.Step(`^the perennial branches are now configured as "([^"]+)"$`, func(name string) error {
+		actual := fs.activeScenarioState.gitEnvironment.DeveloperRepo.FreshConfiguration().GetPerennialBranches()
+		if len(actual) != 1 {
+			return fmt.Errorf("expected 1 perennial branch, got %q", actual)
+		}
+		if actual[0] != name {
+			return fmt.Errorf("expected %q, got %q", name, actual)
```

Comment:
Should this use `actual[0]` in the error message?

Reply:
Yes, good catch!